projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -759,16 +772,176 @@ function wpcom_add_new_page_check( $post_id, $post ) {
+		$about_page_id = (int) $about_page_id;
+		if ( $about_page_id > 0 ) {
+			return $about_page_id;
+		}
+
+		return null;
+	}
+
+	// Call the current function implementation without any caching logic.
+	$about_page_id = wpcom_find_site_about_page_id(); // could also be wpcom_identify_site_about_page_id() or similar
+
+	if ( null === $about_page_id ) {
+		// In the event that we don't have a page id, cache -1 to avoid caching null
+		$value_to_cache = -1;
+	} else {
+		$value_to_cache = $about_page_id;
+	}
+
+	// Use wp_cache_add() to avoid cache stampedes, and specify an expiration time
+	wp_cache_add( $about_page_id_cache_key, $value_to_cache, 'jetpack_mu_wpcom', 3600 );
+
+	return $about_page_id;
+}
+
+/**
+ * Determine if the site has an 'About' page.
+ * We do this by loading the `en` annotation for the site and theme.
+ * We then check if there is a post with the title 'About'. If there is, we get the `hs_old_id` and check to make sure a corresponding post exists in the user's locale.
+ *
+ * @return int|null The page ID of the 'About' page if it exists, null otherwise.
+ */
+function wpcom_find_site_about_page_id() {
+	$annotation = wpcom_get_theme_annotation( get_stylesheet() );
+
+	// Return null if there is no annotation or the annotation doesn't have any content.
+	if ( ! $annotation || ! isset( $annotation['content'] ) || ! is_array( $annotation['content'] ) ) {
+		return null;
+	}
+
+	// Use the annotation to build up a list of 'About' pages.
+	$headstart_about_pages = array_filter(
+		$annotation['content'],
+		function ( $page ) {
+			if ( 'page' !== $page['post_type'] ) {
+				return false;
+			}
+
+			if ( 'about' === $page['post_name'] || false !== strpos( $page['post_title'], 'About' ) ) {
+				return true;
+			}
+		}
+	);
+
+	// Return null if there are no 'About' pages.
+	if ( empty( $headstart_about_pages ) ) {
+		return null;
+	}
+
+	// Get the hs_old_ids for the 'About' pages. We'll use these to find any published pages with the same hs_old_id.
+	$headstart_about_page_hs_old_ids = array_map(
+		function ( $about_page ) {
+			return $about_page['hs_old_id'];
+		},
+		$headstart_about_pages
+	);
+
+	// Return null if there aren't any hs_old_ids to check.
+	if ( empty( $headstart_about_page_hs_old_ids ) ) {
+		return null;
+	}
+
+	$filters     = array(
+		'post_type'    => 'page',
+		'numberposts'  => 1,
+		'fields'       => 'ids',
+		'orderby'      => 'meta_value',
+		'meta_key'     => '_hs_old_id',
+		'meta_compare' => 'IN',
+		'meta_value'   => $headstart_about_page_hs_old_ids,
+	);
+	$about_pages = get_posts( $filters );
+
+	// Return null if we couldn't find any published pages matching the hs_old_ids.
+	if ( empty( $about_pages ) ) {
+		return null;
+	}
+
+	// Cache to hs_old_ids so we can validate the post being updated later.
+	wp_cache_set( 'headstart_about_page_hs_old_ids', $headstart_about_page_hs_old_ids, 'jetpack_mu_wpcom' );
+
+	// Return the id of the first About page.
+	return $about_pages[0];
```

Comment:
Minor thought: it may be worth forcing the returned value to be an integer, just to be _super-defensive_, and ensuring our upstream code can use strict equality comparisons.
```suggestion
	// Return the id of the first About page.
	return (int) $about_pages[0];
```